django_cas_ng/decorators.py

```diff
@@ -32,7 +33,7 @@ def wrapper(request, *args, **kwargs):
             if test_func(request.user):
                 return view_func(request, *args, **kwargs)
             elif request.user.is_authenticated():
-                return HttpResponseForbidden('<h1>Permission denied</h1>')
+                raise PermissionDenied
```

Comment:
Should'nt be `raise PermissionDenied()` here: i.e. instanciate the exception PermissionDenied ?